cmd/bridge/main.go

```diff
@@ -413,6 +424,15 @@ func main() {
 
 		srv.TerminalProxyTLSConfig = serviceProxyTLSConfig
 
+		if strings.TrimPrefix(*fK8sModeOffClusterGitOps, "https://") != "" {
```

Comment:
Remove the `https://` check. We shouldn't be changing product code to workaround an incorrect development script. Let's fix the script.

Reply:
We *definitely* want to fail if someone ever passes `https://` as the value in a real environment.

Reply:
Removed the check for `https://`